src/type/preload.ts

```diff
@@ -640,3 +644,12 @@ export interface MessageBoxReturnValue {
 }
 
 export const SandboxKey = "electron" as const;
+
+/** プロジェクト一時ファイル */
+export type TempProjectType = {
+  appVersion?: string;
+  audioKeys?: AudioKey[];
+  audioItems?: Record<AudioKey, AudioItem>;
```

Comment:
この辺り`ProjectType`と同じであるべきだと思うので、こうでも良いかも。
```suggestion
  project: ProjectType,
```